### PR TITLE
Chore: update documents and add inline comments

### PR DESCRIPTION
**Pull request by zdtsw**
- fix wrong short argument for get_release_issues.py
	- rename from --fix-version to --release in move_to_qa.py
	- add default arguments for get_release_issues.py
	- add post check for after transition status in move_to_qa.py
	- add github action for pylint + fix linter

Signed-off-by: Wen Zhou <wenzhou@redhat.com>

### CODE REVIEW

**Comment by zdtsw** [comment]
@accorvin mind for a review?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
-# RHODS-Jira-Tools
+# RHODS-JIRA-Tools
 
-A collection of tools for managing Jira issues for the RHODS project
+A collection of tools for managing JIRA issues for the RHODS project
```
Comment (accorvin) [inline README.md:3]: This is very much a nit, but I think capitalizing Jira is actually incorrect. It's consistently written as "Jira" on Atlassian's website: https://www.atlassian.com/software/jira

**Comment by accorvin** [comment]
I disagree with the decision to replace single quotes with double quotes here. Most python developers I've worked with and project I've worked on have defaulted to single quotes. I don't have much of a stake in this code base anymore so I'll defer to you and the rest of the DevOps team if you want to switch to this convention, but I think it's worth checking with Deepak and Pablo first.

**Comment by zdtsw** [comment]
> I disagree with the decision to replace single quotes with double quotes here. Most python developers I've worked with and project I've worked on have defaulted to single quotes. I don't have much of a stake in this code base anymore so I'll defer to you and the rest of the DevOps team if you want to switch to this convention, but I think it's worth checking with Deepak and Pablo first.

I was thinking to make a internal fork but then fell maybe others are using it so better to do with a PR.
I will bring this up the question to the other folks and see if we can collect all useful tools in one central place

**Comment by zdtsw** [comment]
some refactor works have been done in a forked repo, close PR

**Pull request closed without merging**